full_install.sh

```diff
@@ -0,0 +1,26 @@
+deactivate
+rm -rf ./venv
+python3 -m venv ./venv
+source ./venv/bin/activate
+pip install -U pip
+pip install mxnet-cu102
```

Comment:
why install gpu mxnet? Why not just assume mxnet is installed prior to calling this script?

Reply:
Text/cv tests won't pass without gpu